src/index.ts

```diff
@@ -47,8 +47,25 @@ function createElementPropertyRecord(
     virtualValue: currentValue,
     mutations: [],
     el,
+    _positionTimeout: null,
     observer: new MutationObserver(() => {
+      // enact a 1 second timeout that blocks subsequent firing of the
+      // observer until the timeout is complete. This will prevent multiple
+      // mutations from firing in quick succession, which can cause the
+      // mutation to be reverted before the DOM has a chance to update.
+      if (attr === 'position' && record._positionTimeout) return;
+      else if (attr === 'position')
+        record._positionTimeout = setTimeout(() => {
+          record._positionTimeout = null;
+        }, 1000);
```

Comment:
Couldn't this cause a mutation to be missed if it happens during that second?

Reply:
It's sort of the intention - the timeout is only set after running the first time, meaning that the mutation has already run and is being run again in less than a second - this acts as sort of a 'cooldown' to prevent each element with its individual mutation observer firing after every position change.